### PR TITLE
Update top-level config files

### LINKED ISSUE #4931: duplicated LICENSE.txt file?
There's one in the top-level directory and one in the statsmodels/ directory.  It looks like the one in statsmodels/ is slightly more up-to-date.

### LINKED ISSUE #4955: .bzrignore needed/used?
Hasn't been touched in 9 years, I'm guessing it's from before the move to github?

### PR DESCRIPTION
**Pull request by jbrockmendel**
For kicks I also enabled doctests, am curious how they do.

Closes #4955 
Closes #4931 
Closes #4677

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,36 @@
+[tool:pytest]
+addopts = --doctest-modules
```
Comment (bashtage) [inline setup.cfg]: You need to drop this.  You won't get doctests to pass in finite time.

**Comment by josef-pkt** [comment]
@jbrockmendel  you can try things locally and try on specific modules or folders first instead of wasting our ci resources. 

I tried running doctest (on my computer) several years ago, and it is useless because almost all our docstrings are written in a way that will not test.
And we are NOT going to change that, at least not in the next 5 years.

File: setup.cfg (hunk 1 above)
Comment (jbrockmendel) [reply setup.cfg]: Will do.  As mentioned, this was just for curiosity sake.

**Comment by jbrockmendel** [comment]
> you can try things locally and try on specific modules or folders first instead of wasting our ci resources.

@josef-pkt I look forward to getting back in this habit just as soon as CI turnaround time comes without a mile of being the binding constraint on the workflow.

**Comment by josef-pkt** [comment]
> I look forward to getting back in this habit just as soon as CI turnaround time comes without a mile of being the binding constraint on the workflow.

You should do this now, when we have so many PRs incoming, when PR action is quite, then it is not as relevant, but testing locally is just "proper behavior" for contributors, developers and maintainers.

**Comment by bashtage** [comment]
One reason to avoid experiments in PRs is that it blocks master builds. When merging it is usually a good idea to make sure master is green before merging the next patch so that if a changeset breaks master it can be quickly fixed or abandoned.  

Personal Travis is a good place to run Experiments. Plus you get email notifications of failure/success/error on personal.

**Comment by josef-pkt** [comment]
For me the main advantage of running module or function specific unit tests is the turnaround time during refactoring. The test runs take at most a few seconds, and it can catch incomplete refactoring mistakes very quickly. (plus it's easy to switch to using pdb)
Then Travis and appveyor are left to check for platform and version problems and to check for long distance side effects outside of the immediate, locally tested modules.

**Comment by jbrockmendel** [comment]
>  but testing locally is just "proper behavior"

Point taken, especially seeing as how I'm making a push for Best Practices elsewhere.

But for the love of Cthulthu, can we implement a system so that trivial PRs don't fall through the cracks for weeks and years at a time?

**Comment by jbrockmendel** [comment]
@josef-pkt 
#4954 
#4941
#4933
#4927
#4924 
#4907 
#4906 
#4888
#4875
#4874 
#4868 
#4849 
#4846 
#4677 (bashtage's, 3 months old)
#4506 (4 months old, unmangling tests)
#2652 (three frakkin years)

are all either green or green-as-it-gets and no-brainers.  Going through another unnecessary rebase cycle would waste CI resources, so maybe its worth taking the 30 seconds apiece to verify they do exactly what the titles suggest and merge them.  Or was that complaint disingenuous?

**Comment by jbrockmendel** [comment]
Closing in favor of #4677 + #4972 + #4973

**Pull request closed without merging**